java/src/main/java/com/amido/stacks/workloads/menu/api/v1/CategoryController.java

```diff
@@ -0,0 +1,110 @@
+package com.amido.stacks.workloads.menu.api.v1;
+
+import static org.springframework.http.HttpStatus.OK;
+
+import com.amido.stacks.core.api.annotations.CreateAPIResponses;
+import com.amido.stacks.core.api.annotations.DeleteAPIResponses;
+import com.amido.stacks.core.api.annotations.UpdateAPIResponses;
+import com.amido.stacks.core.api.dto.response.ResourceCreatedResponse;
+import com.amido.stacks.core.api.dto.response.ResourceUpdatedResponse;
+import com.amido.stacks.workloads.menu.api.v1.dto.request.CreateCategoryRequest;
+import com.amido.stacks.workloads.menu.api.v1.dto.request.UpdateCategoryRequest;
+import com.amido.stacks.workloads.menu.commands.DeleteCategoryCommand;
+import com.amido.stacks.workloads.menu.handlers.CreateCategoryHandler;
+import com.amido.stacks.workloads.menu.handlers.DeleteCategoryHandler;
+import com.amido.stacks.workloads.menu.handlers.UpdateCategoryHandler;
+import com.amido.stacks.workloads.menu.mappers.RequestToCommandMapper;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import java.util.UUID;
+import javax.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestAttribute;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(
+    path = "/v1/menu/{id}/category",
+    produces = MediaType.APPLICATION_JSON_VALUE + "; charset=utf-8")
+public class CategoryController {
+
+  private CreateCategoryHandler createCategoryHandler;
```

Comment:
I suspect IntelliJ is telling you all of these can be final?

Reply:
yes done